### ISSUE TITLE
[Cliente 12] Menus não salvando

### ISSUE THREAD

**Issue by Preichmann** [opentibiabr/otservbr-global-archived#235]
**Describe the bug**
Ao relogar todas as janelas que estao abertas(skills,battle,vip) e a barra de hp que fica no topo da tela voltam para a config original, não esta ficando salvo no client 12

1. Logar no cliente 12(branch dev)
2. Abrir abas e mudar a barra de hp para a posicao de baixo(como na print)
3. Relogar com CTRL+G
4. Verificar que todas as abas vem fechadas ao logar e a barra de HP volta para o topo(default)

**Expected behavior**
As configuracoes devem ficar salvas ao relogar/sair do tibia

**Screenshots**
Arrumar as abas assim:
![image](https://user-images.githubusercontent.com/12971964/60759247-619e9780-9ff8-11e9-890a-8bb736b18030.png)
Relogar com CTRL+G
![image](https://user-images.githubusercontent.com/12971964/60759251-69f6d280-9ff8-11e9-9bc5-446e3ed36fec.png)

**Desktop (please complete the following information):**
- Branch Dev
- Client 12
- Hosteado em win 10

**Comment by Preichmann** [comment]
OBS: O problema não parece ser do cliente, pois se o char ficar logado ele volta com a msm config, por exemplo: 
Logo o char e configuro tudo que quero
Saio fechando o cliente no X
Logo novamente no char
as coisas continuam configuradas

**Comment by Aerwix** [comment]
It's server side, we need a special script to save the backpacks/panels, but right now everyone is focused in other things.

Hopefully this feature will be available soon.

**Comment by gerotib** [comment]
Its saving data correctly, but each time you login your character, new folder in client folder "characterdata" is being created, instead of using the one that should be assigned to this character.

**Comment by vinnyzeramoc** [comment]
qual a solução para isso ?

**Comment by gerotib** [comment]
I made it work by changing function that set player id on login from auto incremented playerAutoID++ to this playerAutoID number which was 0x10000000 + guid (with this player gets same id each login, and client load proper files with settings). Then I had to increase this playerAutoID number so getCreatureByID works properly. I dont think this is a great solution, so I didnt created any pr, but it works fine and requires very few code changes. Heres exactly what ive changed:
in player.h:
```
	void setID() final {
			if (id == 0) {
				id = playerAutoID++;
			}
		}
```
to:
```
		void setID() final {
			if (id == 0) {
				if (guid != 0) {
					id = 0x10000000 + guid;
				}
			}
		}
```
in player.cpp:
`uint32_t Player::playerAutoID = 0x10000000;`
to:
`uint32_t Player::playerAutoID = 0x10010000;` < theres a problem, it should be automatically increased to the highest guid in database(0x10000000 + "highest guid") , now it will work to 10k players, but its probably safe to increase it.

Thats where its used:
in game.cpp
```
Creature* Game::getCreatureByID(uint32_t id)
{
	if (id <= Player::playerAutoID) {
		return getPlayerByID(id);
	} else if (id <= Monster::monsterAutoID) {
		return getMonsterByID(id);
	} else if (id <= Npc::npcAutoID) {
		return getNpcByID(id);
	}
	return nullptr;
}
```
Someone more experienced should take a look and actually code it better, but this could be helpful to understand where this problem lies.

[Theres also explanation from otland thread.](https://otland.net/threads/error-player-configuration-on-client-12.266316/#post-2573166)
